captum/optim/_param/image/transform.py

```diff
@@ -250,8 +250,13 @@ def scale_tensor(
         scale_matrix = self.get_scale_mat(scale, x.device, x.dtype)[None, ...].repeat(
             x.shape[0], 1, 1
         )
-        grid = F.affine_grid(scale_matrix, x.size())
-        x = F.grid_sample(x, grid)
+        try:
```

Comment:
Suggestion: I think that we could also do a version check here like we did with `FFTImage` instead of a try-except block.

Reply:
Added version check!